locales.php

```diff
@@ -1787,19 +1902,19 @@ public function GP_Locales() {
 		$zh->native_name = '中文';
 		$zh->lang_code_iso_639_1 = 'zh';
 		$zh->lang_code_iso_639_2 = 'zho';
-		$zh->country_code = '';
 		$zh->slug = 'zh';
 		$zh->nplurals = 1;
 		$zh->plural_expression = '0';
 
 		foreach( get_defined_vars() as $locale ) {
-			$this->locales[$locale->slug] = $locale;
+			$this->locales[ $locale->slug ] = $locale;
 		}
 	}
 
-	public static function instance() {
-		if ( !isset( $GLOBALS['gp_locales'] ) )
+	public static function &instance() {
```

Comment:
It looks like in GlotPress trac that the & is legacy from @nb's original commit -- https://glotpress.trac.wordpress.org/changeset/75 -- and persisted through https://glotpress.trac.wordpress.org/changeset/778